src/main/java/uk/gov/hmcts/reform/juddata/camel/util/JrdTask.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.juddata.camel.util;
+
+import static uk.gov.hmcts.reform.juddata.camel.util.MappingConstants.ERROR_MESSAGE;
+import static uk.gov.hmcts.reform.juddata.camel.util.MappingConstants.IS_EXCEPTION_HANDLED;
+import static uk.gov.hmcts.reform.juddata.camel.util.MappingConstants.SCHEDULER_STATUS;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.camel.CamelContext;
+import org.apache.camel.ProducerTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.juddata.camel.service.AuditProcessingService;
+import uk.gov.hmcts.reform.juddata.camel.service.EmailService;
+
+@Slf4j
+@Component
+public class JrdTask {
+
+    @Autowired
+    CamelContext camelContext;
+
+    @Autowired
+    DataLoadUtil dataLoadUtil;
+
+    @Autowired
+    ProducerTemplate producerTemplate;
+
+    @Autowired
+    AuditProcessingService auditProcessingService;
+
+    @Autowired
+    EmailService emailService;
+
+    public void execute(CamelContext camelContext, String schedulerName, String route) {
+        try {
+            camelContext.getGlobalOptions().remove(IS_EXCEPTION_HANDLED);
```

Comment:
camelContext.getGlobalOptions() can be stored in variable and can be reused ahead

Reply:
done